### DISCUSSION TITLE
how to search website?

### DISCUSSION THREAD

**Discussion by 0lm** [ImageMagick/ImageMagick#2618] [Help]
hi
i noticed imagemagic forum was renamed by adding "legacy". the search function isnt working anymore because of that. the links google finds still link to the version without "legacy". and since imagemagick forum search feature uses google... im in a loop here. any idea how to search older threads properly? i dont really want to ask the same questions agian, since they were asked lots of times, but i cant access the search results as said. the link pattern seem to have changed completely, because even adding the last part (the view topic part) to the new legacy link pattern wont give me access to the thread. it will just redirect me to the main legacy-forum page.

**Comment by Ian2020** [comment]
This was addressed in the recent [announcement](https://github.com/ImageMagick/ImageMagick/discussions/2565), to quote @urban-warrior:

> In the mean-time, try https://archive.imagemagick.org/discourse-server/viewtopic.php?t=20846, for example.

**Comment by urban-warrior** [comment]
We mirrored the legacy discussion service @ https://legacy.imagemagick.org/discourse-server.  In the mean-time we alerted Google to re-crawl the ImageMagick website to update the links.  The process will likely take several days for Google to export the new links and remove the links that now return 404.  Stability will of course be restored again soon and likely will stay stable for years to come.
